.github/workflows/notify-release.yml

```diff
@@ -15,4 +15,4 @@ jobs:
         id: notify
         uses: nearform/github-action-notify-release@main
         with:
-          notify-after: '0'
+          notify-after: '0 days'
```

Comment:
I think we should fix this in the action. Please **understand** how the ms package work and tweak the action so that whatever the input (as long as it's supported by ms), the action behaves consistently.

ms supports as input both strings and numbers, but the type of the action input will always be a string, therefore that's the only case we need to support. 

We need to be able to support plain numbers (formatted as strings as inputs):

- `'0'` -> 0ms
- `'0ms'` -> 0ms